optimum/bettertransformer/transformation.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2022 The HuggingFace and Meta Team.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from copy import deepcopy
+from typing import Optional
+
+import torch
+
+from optimum.utils import check_if_pytorch_greater_112, is_accelerate_available
+
+from .models import is_module_fast, warn_uncompatible_save
+
+
+if is_accelerate_available():
+    from accelerate import dispatch_model, infer_auto_device_map
+    from accelerate.hooks import remove_hook_from_module
+    from accelerate.utils import named_module_tensors, set_module_tensor_to_device
+
+
+def init_accelerate_hook(module):
+    r"""
+    This function initializes `accelerate` hooks by setting the
+    parameters of the module on the `cpu` if there is any offloading involved
+
+    Args:
+        `module` (`torch.nn.Module`, **required**):
+            The input module to initialize
+    Returns:
+        The initialized module
+    """
+    for name, child in module.named_children():
+        if hasattr(child, "_hf_hook"):
+            if child._hf_hook.weights_map is not None:
+                hook = child._hf_hook
+                if child._hf_hook.offload:
+                    for name, _ in named_module_tensors(
+                        child, include_buffers=hook.offload_buffers, recurse=hook.place_submodules
+                    ):
+                        set_module_tensor_to_device(child, name, "cpu", value=hook.weights_map[name])
+        init_accelerate_hook(child)
+    return module
+
+
+def replace_to_fast(model, config):
```

Comment:
The function should be renamed

Reply:
Fixed in 1c1c6078588fc32f77370fb9b6d24be8bdf45117